index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <h1>"Hello World, We are software developers!"</h1>
+
+    <h1 id="title">Wildlife Survey</h1>
+    <p id="description">This is a survey about wildlife</p>
+    <form id="survey-form">
+      <label for="" id="name-label">Name: </label>
+      <input
+        id="name"
+        type="text"
+        required
+        placeholder="Please write your name"
+      /><br />
+      <label for="" id="email-label"> Email:</label>
+      <input
+        id="email"
+        type="email"
+        required
+        placeholder="Put your email here"
+      />
+      <br />
+      <label for="" id="number-label"> Number: </label>
+      <input
+        type="number"
+        id="number"
+        required
+        min="6"
+        max="10"
+        placeholder="Only write numbers here"
+      /><br />
+      <select id="dropdown">
+        <option value="1">1</option>
+        <option value="2">2</option>
+      </select>
+      <input type="radio" name="names" id="Oscar" value="Oscar" checked />
+      <input type="radio" name="names" id="Habeeb" value="Habeeb" />
+      <br />
+      <input type="checkbox" name="names" id="checkbox" value="oscar" />
+      <input type="checkbox" name="names" id="checkbox" value="Habeeb" />
+      <br />
+      <textarea name="textfield" id="text" cols="30" rows="10"></textarea>
+      <br />
+      <input type="submit" id="submit" value="submit" />
+    </form>
+
+    <div>
```

Comment:
creating html tags that have no use like this one isn't best practice and should be removed before the site is pushed to production.